src/features/zaaksysteem/ZaakZoeker.vue

```diff
@@ -120,12 +136,24 @@ watch(
   () => props.populatedBsn,
   (populatedBsnValue) => {
     if (populatedBsnValue) {
-      bsn.value = populatedBsnValue;
+      store.value.zaakSearchParams.bsn = populatedBsnValue;
       zoekOpBsn();
     }
   },
   { immediate: true }
 );
+
+const singleZaak = computed(() =>
+  store.value.zaken && store.value.zaken.length === 1
+    ? store.value.zaken[0]
+    : undefined
+);
+
+watch(singleZaak, (n, o) => {
+  if (n && n.id !== o?.id) {
+    zaakSelected(n);
+  }
+});
```

Comment:
For me, this would be much easier to read like (at least, I think it's the new and old):

```
watch(singleZaak, (newZaak, oldZaak) => ...
```

Reply:
nemen we mee als we een wijziging in zaakzoeker doorvoeren